source/MessageHandlers/SendDataHandler.ts

```diff
@@ -177,11 +177,14 @@ export default class SendDataHandler implements IMessageHandler {
                     } else {
 
                         try {
```

Comment:
If somebody could check this closely, that would be good. Basically, if the INGEST_RESTORATION_RESOURCES is left out, or set to anything but FALSE/false/FaLsE, it WILL run the queries.